localisation/touhou_events_l_english.yml

```diff
@@ -4297,6 +4297,27 @@
  th_scientistic_flavor.30.a: "Let him study the art of Faith."
  th_scientistic_flavor.30.b: "Let him study the art of Magic."
  th_scientistic_flavor.30.c: "Let him study the art of Science."
+ th_scientistic_flavor.31.t: "Talented Graduate Students"
+ th_scientistic_flavor.31.d: "TODO"
+ th_scientistic_flavor.31.a: "That student will be dedicated to the science of our country."
+ th_scientistic_flavor.31.b: "That student will be dedicated to our exploration."
+ th_scientistic_flavor.31.c: "With such talent should lead our country!"
+ th_scientistic_flavor.32.t: "Fanatic Local Religious Leader"
+ th_scientistic_flavor.32.d: "TODO"
+ th_scientistic_flavor.32.a: "To leave him alone itself means that we are not like other 'religion'."
+ th_scientistic_flavor.32.b: "Will prove our academic superiority if defeat him with arguments."
+ th_scientistic_flavor.32.c: "He's a silly kid, but if you retrain him, he'll be a great evangelist."
+ th_scientistic_flavor.32.d: "Hah! Let's see if his god will protect him through the flames!"
+ th_scientistic_flavor.33.t: "Miniature Games are in Vogue"
+ th_scientistic_flavor.33.d: "TODO"
+ th_scientistic_flavor.33.a: "That will be not just a game, it's a tactical training exercise."
+ th_scientistic_flavor.33.b: "Charging a special sales tax will help our finances."
+ th_scientistic_flavor.33.c: "It's a great opportunity to showcase our technology abroad."
+ th_scientistic_flavor.34.t: "Digging Artifacts in Outworld"
+ th_scientistic_flavor.34.d: "TODO"
```

Comment:
TODOs in the event descriptions...

Reply:
Sorry, I meant to write this later. I've added the necessary descriptions, so please check them out.